src/google/protobuf/compiler/objectivec/objectivec_generator.cc

```diff
@@ -190,19 +190,21 @@ bool ObjectiveCGenerator::GenerateAll(
       // header search path since the generate #import will be more complete.
       generation_options.runtime_import_prefix =
           StripSuffixString(options[i].second, "/");
-    } else if (options[i].first == "default_objc_class_prefix") {
```

Comment:
I can also keep this option.

I figured we can delete it because I dont think a protobuf version has shipped with it.

Reply:
If we drop it, I'd suggesting doin that in a clean PR/commit so git history is a little more clear.

Reply:
sounds good

Reply:
Reverted here in a clean PR https://github.com/protocolbuffers/protobuf/pull/9532